webapp/src/Controller/API/GeneralInfoController.php

```diff
@@ -226,6 +237,7 @@ public function getUserAction()
     /**
      * Get configuration variables
      * @Rest\Get("/config")
+     * @IsGranted("ROLE_ADMIN")
```

Comment:
this shouldn't be here but below at the update code path, right?

Reply:
Yes